src/tpl/CMakeLists.txt

```diff
@@ -147,6 +147,7 @@ if (NOT TARGET fmt::fmt)
     if (NOT EXISTS ${PROJECT_SOURCE_DIR}/src/tpl/umpire/fmt/CMakeLists.txt)
       message(FATAL_ERROR "fmt submodule not initialized. Run 'git submodule update --init --recursive' in the git repository or set fmt_DIR to use an external build of fmt.")
     else ()
+      set(fmt_DIR ${CMAKE_INSTALL_PREFIX} CACHE PATH "")
```

Comment:
If I understand correctly, fmt_DIR is used here: https://github.com/LLNL/Umpire/blob/develop/umpire-config.cmake.in#L37

When using an external fmt the call to find_package will set fmt_DIR for us, which is searched. However, I'm confused why this doesn't work since PACKAGE_CMAKE_INSTALL_PREFIX should be the same as CMAKE_INSTALL_PREFIX: https://github.com/LLNL/Umpire/blob/develop/umpire-config.cmake.in#L45

Reply:
hmm... that is an excellent point. Also explains why I couldn't reproduce the original issue.  I will need to see if I can force this to fail..

man @adayton1 why must you rain on my parade.

Reply:
☔ 🌧️